radio/src/thirdparty/libopenui/src/menu.cpp

```diff
@@ -369,7 +369,7 @@ void Menu::onEvent(event_t event)
 #if defined(HARDWARE_KEYS)
   if (toolbar &&
       (event == EVT_KEY_BREAK(KEY_PAGEDN) ||
-#if defined(KEYS_GPIO_REG_PGUP)
+#if defined(KEYS_GPIO_REG_PAGEUP)
        event == EVT_KEY_BREAK(KEY_PAGEUP)
```

Comment:
@philmoz We should think about replacing these `#if defined(...)` by just simulating the relevant events in a generic fashion. This could be done either at the basic key handling level (`radio/src/keys.cpp`) or in the LVGL driver (`radio/src/gui/colorlcd/LvglWrapper.cpp`).